NETCORE/test/FunctionalTests.EmptyApp.Tests/FunctionalTest/RequestTelemetryEmptyAppTests.cs

```diff
@@ -19,52 +19,6 @@ public RequestTelemetryEmptyAppTests(ITestOutputHelper output) : base (output)
             this.assemblyName = this.GetType().GetTypeInfo().Assembly.GetName().Name;
         }
 
-        [Fact]
-        public void TestBasicRequestPropertiesAfterRequestingBasicPage()
-        {
-            using (var server = new InProcessServer(assemblyName, this.output))
-            {
-                const string RequestPath = "/";
-
-                var expectedRequestTelemetry = new RequestTelemetry();
-                expectedRequestTelemetry.Name = "GET /";
-                expectedRequestTelemetry.ResponseCode = "200";
-                expectedRequestTelemetry.Success = true;
-                expectedRequestTelemetry.Url = new System.Uri(server.BaseHost + RequestPath);
-
-                Dictionary<string, string> requestHeaders = new Dictionary<string, string>()
```

Comment:
do we need to check the request headers in the new tests?

Reply:
new IntegrationTest already has tests for correlation headers.